libs/react-components/src/tedi/components/form/textarea/textarea.tsx

```diff
@@ -13,17 +13,18 @@ export interface TextAreaProps extends TextFieldProps {
 }
 
 export const TextArea = forwardRef<TextFieldForwardRef, TextAreaProps>((props, ref): JSX.Element => {
-  const { className, helper = [], characterLimit, onChange, ...rest } = props;
-  const [value, setValue] = React.useState<string>(props.value ?? '');
-
-  const handleInputChange = (inputValue: string) => {
-    setValue(inputValue);
-
-    if (onChange) {
-      onChange(inputValue);
-    }
-  };
+  const { className, helper = [], characterLimit, onChange, value: externalValue, defaultValue, ...rest } = props;
+  const [innerValue, setInnerValue] = React.useState(externalValue ?? defaultValue ?? '');
```

Comment:
```suggestion
  const [innerValue, setInnerValue] = React.useState(defaultValue ?? '');
```
If innerValue is used the externalValue should not be used.

Reply:
Removed externalValue from initial state.